### PR TITLE
Add baseline test plan for grunt

### PR DESCRIPTION
**Pull request by alexandrasp**
Include baseline test plan for grunt.
Depends on #1313

### CODE REVIEW

**Comment by alexandrasp** [comment]
@nuclearcat @gctucker as we talked early today, this not working yet. The job template is generated as expected, but now we having problems to run. I did some tests using `block_device` as detect and mmcblk1 and I got the kind of same error. 
[mmcblk1](https://lava.collabora.dev/scheduler/job/6869684#L4767) 
[detect](https://lava.collabora.dev/scheduler/job/6870060)

File: config/core/test-configs-chromeos.yaml [hunk 1]
```diff
@@ -170,7 +170,12 @@ device_types:
         flash_tarball: 'cros-flash-fixup.tar.gz'
         flash_tarball_compression: 'gz'
         tast_tarball: 'tast.tgz'
-      block_device: detect
+      reference_kernel:
+        image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-grunt/20220711.0/amd64/bzImage'
+        modules: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-grunt/20220711.0/amd64/modules.tar.xz'
+      filters:
```
[the remark this answers is not in the document]
Comment (alexandrasp) [reply config/core/test-configs-chromeos.yaml]: thanks @nuclearcat

File: config/core/test-configs-chromeos.yaml [hunk 2]
```diff
@@ -148,14 +148,29 @@ device_types:
...
-  #   params:
-  #     block_device: detect
+  hp-11A-G6-EE-grunt_chromeos:
+    base_name: hp-11A-G6-EE-grunt
+    mach: x86
+    arch: x86_64
+    boot_method: depthcharge
+    params: &grunt-params
+      cros_flash_nfs:
+        base_url: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-grunt/20220711.0/amd64/'
```
Comment (nuclearcat) [inline config/core/test-configs-chromeos.yaml:160]: This one need to be set:
        base_url: 'https://storage.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220718.0/amd64/'
similar to octopus and hatch

Comment (alexandrasp) [reply config/core/test-configs-chromeos.yaml:160]: thanks @nuclearcat. Updated.

**Comment by gctucker** [comment]
@alexandrasp Please rebase now that #1313 has been merged.  Also please make it "ready for review" if it's ready.

**Comment by alexandrasp** [comment]
> @alexandrasp Please rebase now that #1313 has been merged. Also please make it "ready for review" if it's ready.

updated @gctucker. Soon I will put here some links to test result.

**Comment by gctucker** [comment]
The issue with some Chrome OS results not appearing on the dashboard still hasn't been fixed so it's tricky to confirm this was tested correctly on staging.  We'll take a look again later as the issue needs to be fixed today.

File: config/core/test-configs-chromeos.yaml [hunk 3]
```diff
@@ -240,8 +243,9 @@ device_types:
 test_configs:
 
   - device_type: hp-11A-G6-EE-grunt_chromeos
-    test_plans: []
-
+    test_plans:
+      - cros-baseline
+      - cros-baseline-fixed
   - device_type: asus-C436FA-Flip-hatch_chromeos
```
Comment (nuclearcat) [inline config/core/test-configs-chromeos.yaml:249]: Please add space between device_type

File: config/core/test-configs-chromeos.yaml [hunk 4]
```diff
@@ -240,8 +243,9 @@ device_types:
 test_configs:
 
   - device_type: hp-11A-G6-EE-grunt_chromeos
-    test_plans: []
-
+    test_plans:
```
Comment (nuclearcat) [inline config/core/test-configs-chromeos.yaml:246]: Please add cros-tast and cros-tast-fixed as on other devices

**Review by nuclearcat — CHANGES_REQUESTED** [review]

File: config/core/test-configs-chromeos.yaml (hunk 3 above)
Comment (mgalka) [reply config/core/test-configs-chromeos.yaml:249]: @nuclearcat Do you mean new line?

Comment (nuclearcat) [reply config/core/test-configs-chromeos.yaml:249]: yes

**Review by nuclearcat — APPROVED** [review]
LGTM, did tested on kernel definitions by hand - it is generating tests for grunt

**Pull request merged**